src/main/java/com/example/librarymanagement/models/entities/Transaction.java

```diff
@@ -0,0 +1,34 @@
+package com.example.librarymanagement.models.entities;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDate;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "transaction")
+public class Transaction {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private LocalDate date_of_borrowing;
```

Comment:
```suggestion
    private LocalDate date_of_borrowing;
```

please follow Java field naming convention and use camel case please.
You could utilize `@Column` annotation for column name definition